index.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Udemy</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/4b9cd3dbd3.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="main-content" id="">
+      <header>
+        <div class="top-header">
+          <div class="bar">
+            <div class="text">
+              <span class="presentation">
+                <span class="title">Courses on sale from E£169.99</span>
+                <span>
+                  | Thanks for enrolling in a free course. Now, upgrade your
+                  learning.
+                </span>
+              </span>
+              <span class="un-title"> Ends in 5h 52m 34s.</span>
+            </div>
+          </div>
+          <button class="remove-text">x</button>
+        </div>
+        <nav class="row">
+          <div class="btn-menu">
+            <span>
+              <i class="fa-solid fa-bars"></i>
+            </span>
+          </div>
+          <a href="/index.html" class="logo">
+            <img src="./images/logo-udemy.svg" alt="udemy logo" />
+          </a>
+          <a href="/" class="category"> Categories </a>
+          <div class="search-bar">
+            <label class="search-label">Search for anything</label>
+            <form action="" class="search-form">
+              <input class="search-input" type="text" name="" id="" />
+              <button class="btn">
+                <i class="fa-solid fa-magnifying-glass"></i>
+              </button>
+            </form>
+          </div>
+          <ul class="list">
+            <li><a href="/" class="bussiness"> Udemy Bussiness </a></li>
+            <li><a href="/" class="teach"> Teach on Udemy </a></li>
+            <li>
+              <a href="/" class="">
+                <i class="fa-solid fa-cart-shopping"></i>
+              </a>
+            </li>
+
+            <li><a href="/" class="login"> Login </a></li>
+            <li><a href="/" class="signup"> Signup </a></li>
+            <li>
+              <a href="/" class="global-icon"
+                ><i class="fa-solid fa-globe"></i>
```

Comment:
`global-icon` shouldn’t be a link right?

Reply:
done